doc/API.md

```diff
@@ -0,0 +1,139 @@
+# Minisafe API
+
+`minisafe` exposes a [JSON-RPC 2.0](https://www.jsonrpc.org/specification)
+interface over a Unix Domain socket.
+
+Note that all addresses are bech32-encoded *version 0* native Segwit `scriptPubKey`s.
+
+| Command                                                     | Description                                          |
+| ----------------------------------------------------------- | ---------------------------------------------------- |
+| [`help`](#help)                                             | Display all available commands                       |
+| [`stop`](#stop)                                             | Stops the minisafe daemon                            |
+| [`getinfo`](#getinfo)                                       | Display general information                          |
+| [`getaddress`](#getaddress)                                 | Get an address                                       |
+| [`listutxos`](#listutxos)                                   | Display a paginated list of utxos                    |
```

Comment:
Can we be more generalistic? Maybe a `gethistory`? Or `listtransactions`?

Ack for pagination.

Reply:
We need both, a listutxos that help the gui to calculate the current balance and say which coins need cycling and a gettransactions or gethistory to see what happened

Reply:
So pagination for gethistory/gettransactions, but not for listutxos (my bad)

Reply:
We could also add filtering to `listutxos` eventually.

nit: how about naming it `listcoins`?